.github/workflows/release-python-client.yaml

```
@@ -0,0 +1,55 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
```

If you are able to use the input/output from the dev-image-update's workflow dispatch, then you'll also need this!
```suggestion
name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
inputs:
  current_semver:
    description: "Latest version of data repo"
    required: true
```